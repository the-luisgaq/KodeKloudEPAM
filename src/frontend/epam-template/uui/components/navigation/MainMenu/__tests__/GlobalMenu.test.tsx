import React from 'react';
import { render } from '@epam/uui-test-utils';
import { GlobalMenu } from '../GlobalMenu';

describe('GlobalMenu', () => {
    it('should be rendered correctly', () => {
        const { asFragment } = render(<GlobalMenu />);
        expect(asFragment()).toMatchSnapshot();
    });
});
