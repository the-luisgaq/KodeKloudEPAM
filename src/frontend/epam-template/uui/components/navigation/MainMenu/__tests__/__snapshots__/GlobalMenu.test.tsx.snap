// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`GlobalMenu should be rendered correctly 1`] = `
<DocumentFragment>
  <button
    class="globalMenuBtn"
    id="global_menu_toggle"
  >
    <div
      class="container uui-icon uui-enabled globalMenuIcon"
    >
      <svg
        class=""
        height="36"
        width="36"
      />
    </div>
  </button>
</DocumentFragment>
`;
